README.md

```diff
@@ -8,7 +8,8 @@
 Defaults for Fastify that everyone can agree on™.<br>
 This plugin adds some useful utilities to your Fastify instance, see the API section to learn more.
 
-Supports Fastify versions `3.x`.
+Supports Fastify versions `4.x`.
+Please refer to [this branch](https://github.com/fastify/fastify-sensible/tree/4.x) and related versions for Fastify `3.x` compatibility.
```

Comment:
Should this be https://github.com/fastify/fastify-sensible/tree/3.x?

Reply:
no because current release is on the v4.x line for Fastify v3, we renamed the module.

Reply:
Gotcha